HeatStroke/Services/MTL/MTLFile.cpp

```diff
@@ -0,0 +1,164 @@
+//================================================================================
+// MTLFile.cpp
+// Author: David Hanna
+// 
+// Loads an MTL file into a collection of data that may be used by other objects.
+//================================================================================
+
+#include "MTLFile.h"
+
+std::string HeatStroke::MTLFile::MTLMaterial::ToString() const
```

Comment:
outside of testing purposes, is there much point of a ToString function for MTLFile objects?